app/vite.config.ts

```diff
@@ -48,6 +49,6 @@ export default defineConfig({
     build: {
         target: "es2015",
         sourcemap: true,
-        minify: true,
+        minify: env.VITE_BYPASS_MINIFY === "false",
```

Comment:
Change to the following to ensure that it always runs (even without setting the environmental variable) unless set to true:
```suggestion
        minify: env.VITE_BYPASS_MINIFY !== "true",
```